### PR TITLE
Plugins disabler mu-plugin

### PR DESCRIPTION
**Pull request by lusimeon**
Add the ability to disable plugins depends on environment (local,preprod,production).

The following server environment variables must be set:
```bash
DISABLE_PLUGINS_[ENVIRONMENT]=example/example1.php
```

### CODE REVIEW

File: template/web/wp-content/mu-plugins/studiometa-plugin-disabler/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Studio Meta plugin disabler
+
+Disable plugins depends on environment.
+
+## Usage
+
+The following server environment variables must be set:
+```bash
+DISABLE_PLUGINS_[ENVIRONMENT]=example/example1.php
```
Comment (titouanmathis) [inline template/web/wp-content/mu-plugins/studiometa-plugin-disabler/README.md:9]: **suggestion:** Only use the plugin's folder name here

Unless there is a specific interest in having both the folder and the main PHP file name?

**Review by titouanmathis — CHANGES_REQUESTED** [review]
Nice addition!

Could you add the plugin path to both the PHPCS config in `phpcs.xml` and the PHPStan config in `phpstan.neon`?

File: template/web/wp-content/mu-plugins/studiometa-plugin-disabler/README.md (hunk 1 above)
Comment (lusimeon) [reply template/web/wp-content/mu-plugins/studiometa-plugin-disabler/README.md:9]: Plugins aren't always based on the `plugin_name/plugin_name.php` naming rule. Sometimes, they have odd naming structure.

**Review by titouanmathis — APPROVED** [review]
👍

**Comment by titouanmathis** [comment]
@lusimeon Could you add a small section in the template's README.md file about this plugin?

**Review by titouanmathis — APPROVED** [review]

**Review by titouanmathis — APPROVED** [review]

File: template/web/wp-content/mu-plugins/autoload.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
```
Comment (titouanmathis) [inline template/web/wp-content/mu-plugins/autoload.php]: The [boxuk/wp-muplugin-loader](https://github.com/boxuk/wp-muplugin-loader) package handles the autoloading of MU plugins in directories, this file is not needed.

**Review by titouanmathis — CHANGES_REQUESTED** [review]

**Review by titouanmathis — APPROVED** [review]

**Pull request merged**